src/player.js

```diff
@@ -23,8 +23,8 @@ class Player {
      */
     constructor(element, options = {}) {
         /* global jQuery */
-        if ('jQuery' in window && element instanceof jQuery) {
```

Comment:
Perhaps, but checking jQuery in window is just pointless. Its bug prone and just shouldn't be used.